docs/user-guide/faq.rst

```diff
@@ -0,0 +1,92 @@
+FAQs
+====
+
+**How Pulp determines uniqueness when storing units?**
+
+Each type of content unit has a field called 'unit_key_fields' which is
+a nothing but a combination of some fields that uniquely identify
+that particular content unit.
+
+**DRPM**:
+
+- epoch
+- version
+- release
+- filename
+- checksumtype
+- checksum
+
+**RPM, SRPM**:
+
+- name
+- epoch
+- version
+- release
+- arch
+- checksumtype
+- checksum
+
+**ISO**:
+
+- name
+- checksum
+- size
+
+**Python package**:
+
+- filename
+
+**Puppet module**:
+
+- author
+- name
+- version
+
+**OSTree branch**:
+
+- remote_id
+- branch
+- commit
+
+**Docker Blob, Manifest, ManifestList**:
+
+- digest
+
+**Docker Image**:
+
+- image_id
+
+**Docker Tag**:
+
+- name
+- repo_id
+- schema_version
+- manifest_type
+
+**Debian Package**:
+
+- name
+- version
+- architecture
+- checksumtype
+- checksum
+
+**What happens when a source repository changes the checksum type that is
+published in the repository?**
+
+Some of the content types, especially RPM, supports many checksum types while
+publishing a repository. Since, this checksum is a part of the
+'unit_key_fields', it is possible for a repository to have multiple copies of
+the same content unit. Therefore, you need to be careful when changing the
+checksum type while publishing.
+
+**How Pulp keeps track of units that belong to a particular repository?**
```

Comment:
Each repository is stored as a document in the 'repos' MongoDB collection. Each content type is stored in a collection with a prefix of 'units_'. Relationships between content and repositories are stored in the 'repo_content_units' collection.